modin/experimental/engines/omnisci_on_ray/io.py

```diff
@@ -186,6 +186,9 @@ def read_csv(
                 null_values=None,
                 true_values=None,
                 false_values=None,
+                # timestamp fields should handled as strigs if they don't passed
```

Comment:
Grammar: timestamp fields should be handled as strings it they are not passed ....

Reply:
Fixed.